### ISSUE TITLE
ownCloudUserManual.pdf default file

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/example-files#5]
There are some issues with the »ownCloudUserManual.pdf« default file we should fix:

* The name »ownCloudUserManual.pdf« can be improved. Should be called »ownCloud Manual.pdf« for better readability.
* It is set with Helvetica for headings and serifs for body text. According to our styles we should use Open Sans everywhere instead.
* There are some horizontal lines throughout the document. These we can just remove as they make everything look old-style. Whitespace is enough to separate.

@karlitschek whom do I need to talk to about this? :) @carlaschroder you maybe because you work on the documentation?

cc @owncloud/designers

**Comment by karlitschek** [comment]
The filename is now fixed in the next build.
The expert for the theme in the pdf is @danimo

**Comment by jancborchardt** [comment]
Cool, thanks!

@danimo let me know how I can help with that. :)

**Comment by danimo** [comment]
@jancborchardt Good luck messing with LaTeX...

**Comment by jancborchardt** [comment]
@danimo well I’m only asking for 2 things:
* change typeface from Helvetica and the serif to just Open Sans
* remove the horizontal lines

That can’t be that difficult even with LaTeX ;)

**Comment by jancborchardt** [comment]
Let me rephrase:
* `s/Helvetica/Open Sans/g` and `s/that serif font/Open Sans/g`
* `s/horizontal line//g`

I guess :D

**Comment by carlaschroder** [comment]
Sorry but I can't agree with the name change. We have a user manual and an admin manual.

**Comment by jancborchardt** [comment]
@carlaschroder ok, can we then at least call it »ownCloud user manual.pdf«? I’m not too keen on having the word »user« float around in the interface, but if it’s absolutely necessary then please at least with some spaces.

**Comment by carlaschroder** [comment]
@jancborchardt Of course it's necessary, unless you think people being confused is good UI :) I'll poke around with the name when I have a minute. Not sure if that's a label, or the actual filename. If it's the filename then there may be unintended consequences from having spaces in the filename.

**Comment by plastikschnitzer** [comment]
Hi, as far as I see, this file also does not get updated. So in my case I started with ownCloud 6 and in my root folder, there is still the ownCloudusermanual.pdf from version 6 two years ago (now using ownCloud 9) :-D

**Comment by carlaschroder** [comment]
Closing for age.
